### PR TITLE
AC-1019 show last login date

### PR DESCRIPTION
**Pull request by lillie-dae**
### Description

Fixes # .

A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [ ] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New unit tests added to cover the changes.
- [ ] New e2e tests added to cover the changes. (If it has been decided these will not be added with the PR, a separate ticket **must** be created and linked in the ticket below before merge)
- [ ] All tests passed locally.

### CODE REVIEW

**Review by mehranrashid — APPROVED** [review]

File: src/utils/date-utilities.ts [hunk 1]
```diff
@@ -20,13 +20,14 @@ import moment from "moment";
...
+export const formatDateAndTimeOfArray = <T>(
+    items: T[],
+    itemProperty: keyof T,
     inludeFromNow = true,
 ): void => {
-    items.map((item: any) => {
-        item[itemProperty] = formatDateAndTimeOfString(item[itemProperty], inludeFromNow);
+    items.map((item: T) => {
+        const value = item[itemProperty] as string;
+        item[itemProperty] = formatDateAndTimeOfString(value, inludeFromNow) as T[keyof T];
```
Comment (rowellx68) [inline src/utils/date-utilities.ts:30]: can we instead let the UI format the date using filters?

File: src/components/user-management/UserTabItem.vue [hunk 2]
```diff
@@ -96,6 +96,12 @@
                                     {{ item.enabled ? "Enabled" : "Disabled" }}
                                 </v-chip>
                             </td>
+                            <td
+                                class="text-start text-no-wrap"
+                                :data-cy="`user-table-row-email-${index}`"
+                            >
+                                {{ item.lastLoginDate }}
```
Comment (rowellx68) [inline src/components/user-management/UserTabItem.vue:103]: we could use a filter here instead `{{ item.lastLoginDate | filterNameHere }}`

File: src/components/user-management/UserTabItem.vue [hunk 3]
```diff
@@ -96,6 +96,12 @@
                                     {{ item.enabled ? "Enabled" : "Disabled" }}
                                 </v-chip>
                             </td>
+                            <td
+                                class="text-start text-no-wrap"
+                                :data-cy="`user-table-row-email-${index}`"
```
Comment (rowellx68) [inline src/components/user-management/UserTabItem.vue:101]: `user-table-row-last-login-${index}`

File: src/utils/date-utilities.ts (hunk 1 above)
Comment (lillie-dae) [reply src/utils/date-utilities.ts:30]: I've updated place were its happening but leaving this change in as it just adds typing too formatDateAndTimeOfArray function itemProperty.

**Review by rowellx68 — APPROVED** [review]

**Pull request merged**